internal/pkg/debug/render/render_test.go

```diff
@@ -30,24 +30,33 @@ import (
 	"golang.org/x/tools/go/ssa/ssautil"
 )
 
+func TestDOT(t *testing.T) {
+	testGolden(t, DOT, "DOT", ".dot")
+}
+
 func TestSSA(t *testing.T) {
+	testGolden(t, SSA, "SSA", ".ssa")
+}
+
+func testGolden(t *testing.T, fn func(f *ssa.Function) string, fnName, ext string) {
```

Comment:
What is "golden". Maybe add some comment?

Reply:
A "golden" file is a file that contains the expected output for a test. I've seen/used that term many times, so I assumed just "golden" would be clear enough. After doing some digging though, it seems that this term is actually [not that common](https://softwareengineering.stackexchange.com/questions/358786/what-is-golden-files).

Renamed to `testGoldenFiles` and added a comment.